gap/oper.gi

```diff
@@ -530,6 +530,16 @@ function(D, u, v)
 
 end);
 
+InstallMethod(DigraphContractEdge,
+"for a digraph and a list",
+[IsDigraph, IsList],
+function(D, edge)
+  if Length(edge) <> 2 then
```

Comment:
Shouldn't this also check `IsDenseList`?  In case, the second entry of edge is bound but the first not.